packages/abi-gen-wrappers/src/generated-wrappers/dutch_auction.ts

```diff
@@ -511,6 +511,7 @@ export class DutchAuctionContract extends BaseContract {
             return abiEncodedTransactionData;
         },
     };
+
```

Comment:
is this not a tslint violation? i know we've been saying "yes run prettier!" as part of abi-gen. but maybe we should also be running generated code through tslint too? :man_shrugging: maybe, maybe not... it seems like the right thing to do (but certainly not in this PR of course!) but it will likely create a not insignificant pile of work for us...

Reply:
This really bugged me! I ran it through `lint` and `prettier` but the whitespace line stayed. Our style rule enforces max of 1 line between method (I recall discussing this in an eng meeting a couple months ago, too!), but having **no** line between methods is **not** disallowed. Having no line between methods seems to be an `abi-gen-wrappers` thing, most other files have one line between methods. 

In any case, this whitespace is removed in the next commit. oops ^^